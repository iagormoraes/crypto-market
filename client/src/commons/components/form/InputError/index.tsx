import React from 'react';

export default function InputError({
  children,
}: {
  children: React.ReactNode;
}) {
  return <div className="text-red-600 font-bold mb-2">{children}</div>;
}
