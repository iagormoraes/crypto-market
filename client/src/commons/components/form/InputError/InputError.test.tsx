import { render } from '@testing-library/react';

import InputError from '.';

describe('<InputError/>', () => {
  it('matches snapshot', () => {
    const { container } = render(<InputError>Error</InputError>);

    expect(container).toMatchSnapshot();
  });
});
