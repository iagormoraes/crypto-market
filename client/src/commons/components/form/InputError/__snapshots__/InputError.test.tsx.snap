// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputError/> matches snapshot 1`] = `
<div>
  <div
    class="text-red-600 font-bold mb-2"
  >
    Error
  </div>
</div>
`;
